src/lib/types.ts

```diff
@@ -23,8 +23,8 @@ export interface KeyValueString {
 export interface KeyValueNumberString {
     [s: number]: string;
 }
-// eslint-disable-next-line
 export interface KeyValueAny {
+    // eslint-disable-next-line
```

Comment:
`eslint-disable-next-line @typescript-eslint/no-explicit-any`
Best to be specific instead of disabling all. Might be other places.

Reply:
Good point, also removed some